### PR TITLE
Add FreeBSD 14.0-RELEASE

### PR DESCRIPTION
**Pull request by lwhsu**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have run the pre-merge tests locally and they pass.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] If `Gemfile.lock` has changed, I have used `--conservative` to do it and included the full output in the Description above.
- [ ] All new and existing tests passed.
- [ ] All commits have been signed-off for [the Developer Certificate of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

**Review by Stromweld — COMMENTED** [review]
1 minor fix and It looks good.

File: os_pkrvars/freebsd/freebsd-14-aarch64.pkrvars.hcl [hunk 1]
```diff
@@ -0,0 +1,9 @@
+os_name                 = "freebsd"
+os_version              = "14.0"
+os_arch                 = "x86_64"
```
[the remark this answers is not in the document]
Comment (lwhsu) [reply os_pkrvars/freebsd/freebsd-14-aarch64.pkrvars.hcl:3]: Thanks! Then I think `freebsd-13-aarch64.pkrvars.hcl` needs the same fix, while `freebsd-12-aarch64.pkrvars.hcl` is fine.

[the remark this answers is not in the document]
Comment (Stromweld) [reply os_pkrvars/freebsd/freebsd-14-aarch64.pkrvars.hcl]: you are correct. 13 does have the same typo.

**Review by Stromweld — APPROVED** [review]
LGTM

**Pull request merged**